tests/recipe/baked_recipe_test.py

```diff
@@ -55,9 +55,12 @@ def recipe_instances(self):
     def test_from_folder(self, folder):
         parsed_instance = BakedRecipe.from_folder(folder)
 
-        parsed_instance.to_yaml(
-            os.path.join(ASSET_FOLDER, 'baked', f'{parsed_instance.metadata.name}.yaml')
-        )
+        parsed_test_path = os.path.join(ASSET_FOLDER, 'baked', f'{parsed_instance.metadata.name}.yaml')
+
+        if os.path.exists(parsed_test_path):
```

Comment:
Shouldn't this be an assertion?

```suggestion
        assert os.path.exists(parsed_test_path)
```

Reply:
Nah, being able to read it is one thing. Checking whether it compiles correctly means we also have to write the compiled test which is work :smile: